packages/SA/src/inputTools.ts

```diff
@@ -0,0 +1,50 @@
+// inputTools.ts
+
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+import {getGeneratedColumnsData} from './inputsGeneration';
+
+export type VariedNumericalInputInfo = {
+  name: string,
+  caption: string | undefined,
+  type: DG.COLUMN_TYPE,
+  min: number,
+  max: number,
+  column: DG.Column | undefined
```

Comment:
This structure could be simpler. 
`name`, `caption`, `type` could be stored directly in DG.Property

`column` field is always undefined before values generation. It is easier to create separate structure for filled info.
For example,
```
type VariedNumericalInputInfo = {
  prop: DG.Property,
  min: number,
  max: number,
}
```
and 
```
type VariedNumericalInputValues = VariedNumericalInputInfo & {column: DG.Column}
```

What do you think about it?

Reply:
Done